stable/traefik/templates/configmap.yaml

```diff
@@ -92,10 +92,32 @@ data:
     {{- end}}
     format = "{{ .Values.accessLogs.format }}"
     {{- end}}
+    {{- if and .Values.kvprovider.type .Values.kvprovider.endpoint }}
+    [{{ .Values.kvprovider.type }}]
+    endpoint = "{{ .Values.kvprovider.endpoint }}"
+    watch = {{ .Values.kvprovider.watch }}
+    prefix = {{ .Values.kvprovider.prefix }}
+    {{- if eq .Values.kvprovider.type "etcd" }}
+    useAPIV3 = true
```

Comment:
This should be configurable. It is not defaulted to true when using the etcd provider.

Reply:
Yeah absolutely. Very attentive! Fixed it